docs/reference/settings.md

```diff
@@ -441,7 +585,7 @@ For admins only, Wagtail performs a check on the dashboard to see if newer relea
 
 If admins should only be informed of new long term support (LTS) versions, then set this setting to `"lts"` (the setting is case-insensitive).
 
-## Private pages / documents
+## Front-end authentication
```

Comment:
```suggestion
## Fronteend authentication
```
Can we rename this to frontend one word as we use the one word term further down.